src/config.ts

```diff
@@ -1517,4 +1524,42 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx'],
 		explorerUrlToTx: 'https://www.mintscan.io/stargaze/txs/{txHash}',
 	},
+	{
+		rpc: 'https://node0.mainnet.lum.network/rpc',
+		rest: 'https://node0.mainnet.lum.network/rest',
+		chainId: 'lum-network-1',
+		chainName: 'Lum Network',
+		stakeCurrency: {
+			coinDenom: 'LUM',
+			coinMinimalDenom: 'ulum',
+			coinDecimals: 6,
+			coinGeckoId: 'lum-network',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/lum.svg',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('lum'),
+		currencies: [
+			{
+				coinDenom: 'LUM',
+				coinMinimalDenom: 'ulum',
+				coinDecimals: 6,
+				coinGeckoId: 'lum-network',
+				coinImageUrl: window.location.origin + '/public/assets/tokens/lum.svg',
+			},
+		],
+		feeCurrencies: [
+			{
+				coinDenom: 'LUM',
+				coinMinimalDenom: 'ulum',
+				coinDecimals: 6,
+				coinGeckoId: 'lum-network',
+				coinImageUrl: window.location.origin + '/public/assets/tokens/lum.svg',
+			},
+		],
+		coinType: 118,
+		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
```

Comment:
Should there be gas price steps here?

Noticed they are present for $HUAHUA

https://github.com/osmosis-labs/osmosis-frontend/pull/146/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R1560-R1564

Reply:
Yes

Reply:
Do you need me to do something here? We have a min gas pice set by all validators to `minimum-gas-prices = "0.001ulum"`
Not sure on how to interpret this setting in your file though

Reply:
Closing the loop, I think it just informs Keplr "reasonable" values for low/medium/high gas